onboard/catkin_ws/src/controls/scripts/drc_utils.py

```diff
@@ -0,0 +1,71 @@
+from tf.transformations import euler_from_quaternion, quaternion_multiply, quaternion_conjugate
```

Comment:
Lets name this just utils, or controls_utils. drc_utils implies that it is utils used throughout the codebase.

Reply:
fixed in 189a96e